src/main/java/screens/Window.java

```diff
@@ -106,20 +124,27 @@ public static Backlog getBackLog() {
         return backLog;
     }
 
+    /**
+     * Creates instance of InfoScreen.
+     * @return InfoScreen object.
+     */
     public static InfoScreen getInfoScreen() {
         if (infoScreen == null) {
             infoScreen = new InfoScreen();
         }
         return infoScreen;
     }
 
+    /**
+     * Gets the loading screen window.
+     * @return LoadingScreen object.
+     */
     public static LoadingScreen getLoadingScreen() {
         return loadingScreen;
     }
 
     /**
      * Creates the menu bar with its items.
-     *
      * @param stage Main stage.
      * @return The menu bar object.
      */
```

Comment:
This method is rather long, is there any way you can split this into multiple smaller methods?